### PR TITLE
feat(crowdaction): Expandable text for CrowdAction description.

### LINKED ISSUE #190: [Feature]: Truncate description and add More/Less functionality
We will truncate the description on the CrowdAction Details screen to show only up to 200 characters (Substring of description 0, 200), and then append an ellipsis at the end.

When the description is collapsed it will show 200 characters + ellipsis, followed by a "Read more" button that will expand the description to its full body length.

When the description is expanded it will show the full description, followed by a "Show less" button that will collapse the description back to the 200 characters + ellipsis.

**Awaiting Design!**

### PR DESCRIPTION
**Pull request by wizlif**
Closes #190

### CODE REVIEW

File: lib/presentation/shared_widgets/expandable_text.dart [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+import '../themes/constants.dart';
+
+/// Text widget to show read more and read less
+class ExpandableText extends StatefulWidget {
+  const ExpandableText(
+    this.text, {
+    Key? key,
+    this.trimLines = 3,
+    this.clickableTextColor = kAccentColor,
+    this.readMoreText = "... read more",
```
Comment (rubenhorn) [inline lib/presentation/shared_widgets/expandable_text.dart:13]: maybe just "more" and "less"?

**Review by rubenhorn — DISMISSED** [review]

File: lib/presentation/shared_widgets/expandable_text.dart (hunk 1 above)
Comment (Xazin) [reply lib/presentation/shared_widgets/expandable_text.dart:13]: Also the three dots (ellipsis) should not be the same color as the more text right?

Comment (wizlif) [reply lib/presentation/shared_widgets/expandable_text.dart:13]: I was following the design and at least that's what it specifies.

Bildschirmfoto 2022-02-11 um 15 56 06

_Originally posted by @laschnik in https://github.com/CollActionteam/collaction_app/issues/190#issuecomment-1036297382_

Comment (Xazin) [reply lib/presentation/shared_widgets/expandable_text.dart]: Ah yes I see, I like the changes you just committed better haha

**Review by Xazin — APPROVED** [review]

**Pull request merged**